index.css

```diff
@@ -0,0 +1,327 @@
+    /* position: relative; */
+    position:fixed;
+    
+    padding: 2rem;
+    padding-left:4rem;
+    width:100%;
+    padding-right: 4rem;
+    /* box-shadow: 10px 10px 8px 10px #888888; */
+    border:1px solid grey;
+    margin-bottom:3rem;
+    display: block;
+    background-image: linear-gradient(to right, red,orange,yellow,green,blue,indigo,violet);
+    border-radius:15px;
+    text-align: center;
+    z-index: 3;
+
+}
+.categories{
+    width:60%;
+    display: flex;
+    flex-direction: column;
+   text-align:center;
+   gap:1rem;
+   position:absolute;
+   top:10rem;
+   
+   /* background-color: white; */
+}           
+.category{
+    display:flex;
+    flex-direction: column;
+    padding:1rem;
+    width:40%;
+    background-color: red;
+    min-height: 400px;
+    border:1px solid black;
+    left:200px;
+    box-shadow:10px 10px 8px 10px #888888;
+    position:relative;
+    left:42rem;
+    border-radius:10px;
+    overflow: hidden;
+    margin-bottom:1rem;
+    cursor:pointer;
+    justify-content: space-around;
+    gap:1rem;
+
+
+}
+.category a{
+    text-decoration:none;
+    font-size: 2rem;
+    color:white;
+    cursor:pointer;
+    background-color: #612897;
+    border-radius: 10px;
+    padding:1rem;
+}
+.category a:hover{
+    border:none;
+    background-color: #F6830F;
+}
+.category-content{
+    font-size:3rem;
+    background-color: #041C32;
+    color:aliceblue;
+    padding:1rem;
+    border-radius:15px;
+    /* position: absolute;
+    top:20px; */
+    /* padding-top:30%; */
+}
+.cat1{
+    background-color: #D3ECA7;
+    background-image:url('mathbg.jpeg');
+    background-size:cover;
+    
+}
+.cat2{
+    background-color: #655D8A;
+    background-image:url('statesbg.webp');
+    background-size:cover;
+    background-repeat: no-repeat;
+}
+.cat3{
+    background-color: #655D8A;
+    background-image:url('programmingbg.jpeg');
+    background-size:cover;
+    background-repeat: no-repeat;
+}
+.cat4{
+    background-color: #655D8A;
+    background-image:url('politicsbg.jpeg');
+    background-size:cover;
+    background-repeat: no-repeat;
+}
+
+/********************* rule quiz ***************************/
+.rule-quiz{
+    width: 100%;
+    /* text-align: center; */
+
+    
+}
+.rules{
+    background-color: #082032;
+    width:70%;
+    text-align: left;
+    padding:1rem;
+    border-radius:10px;
+    margin-left:16rem;
+    margin-top:10rem;
+    position: absolute;
+    z-index:2;
+    
+}
+.rule-quiz h1{
+    color: aliceblue;
+    font-size: 4rem;
+    font-weight: 600;
+    text-align: center;
+    /* position: relative; */
+    
+    padding: 2rem;
+    padding-left:4rem;
+    width:100%;
+    padding-right: 4rem;
+    /* box-shadow: 10px 10px 8px 10px #888888; */
+    border:1px solid grey;
+    margin-bottom:3rem;
+    display: block;
+    background-image: linear-gradient(to right, red,orange,yellow,green,blue,indigo,violet);
+    border-radius:15px;
+    text-align: center;
```

Comment:
please refactor your code, remove repeated unnecessary code
and use shorthand where ever possible